packages/node_modules/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -213,7 +213,7 @@ class Transforms {
 
         // post and share activities have content which needs to be decrypted
         // as do meeting and recording activities
-        if (!['post', 'share'].includes(activity.verb) && !activity.meeting && !activity.recording) {
+        if (!['post', 'share'].includes(activity.verb) && !activity.meeting && !activity.recording && !activity.extension) {
```

Comment:
we should ensure that it is only for customApp extension type. So you could include somethings like:
!(activity.extension && activity.isCustomApp())

Reply:
Modified condition to ensure for activity extension type is custom app, but referring to isCustomApp() returns undefined value hence used !(activity.extension && activity.extension.extensionType =='customApp')